jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
+```text
+? Would you like to perform an automated setup of this orb?:
+  ▸ Yes, walk me through the process.
+    No, I'll handle everything myself.
+```
 
-`circleci orb source sandbox/hello-world@0.0.1`
+If you would simply like a convenient way of downloading the [Orb-Project-Template](https://github.com/CircleCI-Public/Orb-Project-Template) you can opt to handle everything yourself.
 
-## Designing Orbs
+We will however select to take the automated route. With this option, the [Orb-Project-Template](https://github.com/CircleCI-Public/Orb-Project-Template) will be downloaded and automatically modified with our custom options. After, the project will be automatically followed on CircleCI with an automated CI/CD pipeline included. We'll expend more on the included CI pipeline in our [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs/#section=configuration) documentation.
 
-When designing your own orbs, make sure your orbs meet the following requirements:
+#### 4) You will be asked a short series of questions to configure and setup your orb such as your [namespace]({{site.baseurl}}/2.0/orb-concepts/#namespaces) and orb name.
+{:.no_toc}
 
-* Orbs should always use `description`. - Be sure to explain usage, assumptions, and any tricks in the `description` key under jobs, commands, executors, and parameters.
-* Match commands to executors - If you are providing commands, try to provide one or more executors in which they will run.
-* Use concise naming for your orb - Remember that use of your commands and jobs is always contextual to your orb, so you can use general names like "run-tests" in most cases.
-* Required vs. optional parameters - Provide sound default values of parameters whenever possible.
-* Avoid job-only orbs - Job-only orbs are inflexible. While these orbs are sometimes appropriate, it can be frustrating for users to be unable to use the commands in their own jobs. Pre and post steps when invoking jobs are a workaround for users.
-* Parameter `steps` are powerful - Wrapping steps provided by the user allows you to encapsulate and sugar things like caching strategies and other more complex tasks, providing a lot of value to users.
+In the background the `orb init` command will be copying and customizing the [Orb Project Template](https://github.com/CircleCI-Public/Orb-Project-Template) based on your inputs. There is a detailed `README.md` file within each directory that contains helpful information specific to that directory.
 
-Refer to [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) for details and examples of commands, executors and parameters in orbs.
+You will also be asked for the remote git repository URL which we obtained in step 1. Ensure you are using the `HTTPS` URL.
 
-When developing your own orb, you may find it useful to write an inline orb. The section below describes how you can write your own inline orb.
+#### 5) When prompted, in a separate terminal, push the changes up to Github.
```

Comment:
Does the prompt tell you to use a separate terminal?

Reply:
it does 👍  It also wont let you type or anything so it should be very clear.